ajax/libs/youtube-google-analytics/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "youtube-google-analytics",
+  "filename": "lunametrics-youtube.gtm.min.js",
+  "version": "8.1.0",
+  "description": "This is a plug-and-play tracking solution for tracking user interaction with YouTube videos in Google Analytics. It will detect if GTM, Universal Analytics, or Classic Analytics is installed on the page, in that order, and use the first syntax it matches unless configured otherwise. It include support for delivering hits directly to Universal or Classic Google Analytics, or for pushing Data Layer events to be used by Google Tag Manager.",
+  "homepage": "https://github.com/lunametrics/youtube-google-analytics",
+  "keywords": [
+    "analytics",
+    "youtube",
+    "tracking"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lunametrics/youtube-google-analytics.git",
+    "basePath": "dist",
+    "files": [
+      "lunametrics-youtube.gtm.min.js",
+      "lunametrics-youtube.gtm.js",
+      "luna-youtube-tracking.json"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lunametrics/youtube-google-analytics.git"
+  },
+  "license": "Creative Commons 4.0 International Public License",
```

Comment:
Because this license doesn't exist in [license-list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json), let's modify to this format:
```json
  "license": {
    "type": "Creative Commons 4.0 International Public License",
    "url": "https://github.com/lunametrics/youtube-google-analytics/blob/master/LICENSE.MD"
  }
```